04_environment/index.tex

```diff
@@ -160,26 +145,30 @@ \subsubsection{Incorporated Changes: Population Density}
 
 \subsubsection{Incorporated Changes: Method of Resource Allocation}
 
-Lastly, in future implementations, a parameter can be set that allows either an equal split of returns after foraging no matter the input, or a proportional split that is dependent on the amount of resources inputted by each island. Both of these implementations work in accordance to the previous specification. Such change will allow us to observe the concept of free-riding further, looking at how greedy islands behave in the case of uniform resource returns. Will the islands seek to minimise inputs and benefit from the more generous islands, even at the cost of being allocated a lower tier? Will they input just enough to maximise the tier they are foraging in? Or, will they observe other island’s previous contributions and make a decision based on that? These are some of the questions that can later be explored using this future change.
+Moreover, there is a parameter to allow either an equal split of returns after foraging regardless of the input, or a proportional split dependent on the amount of resources inputted by each island. Both of these implementations work in accordance to the previous specification. Such change will allow us to observe the concept of free-riding further, looking at how greedy islands behave in the case of uniform resource returns.
+
+The distribution method is based on two scalars. The first scales the inputs to the tiers, allowing the agent to choose how much they wish the tiers to cost. After which, the number of fish or deer caught is chosen by the tiers which goes into their respective distributions. The output of these distributions are then multiplied by the output scale. This allows the agent to tweak how much return each island can get, it is suggested to keep the output scalar to above the cost of living to ensure the islands have enough resources to survive. 
```

Comment:
last sentence -- revise :)

suggestion: **... can get. It is ...**

do NOT use the word 'tweak' , sounds informal